api/modules/files/cache.js

```diff
@@ -45,12 +45,11 @@ class RemixedFileCreationCache extends BaseCache {
 
     if (typeof fileBuffer !== `function`) {
       if (!this.cache) {
-        console.log(`NO cache, buffer not set in cache`);
         return next(null, fileBuffer);
       }
 
       return this.cache.setex(`file:${fileId}`, DEFAULT_BUFFER_CACHE_EXPIRY_SEC, fileBuffer)
```

Comment:
Sometimes you set your TTL as `SEC` and other times `MS`.  Is it different for these various APIs?